src/test/gc.test.ts

```diff
@@ -74,6 +74,7 @@ async function retryWithGcUntilCallbackDoesNotThrow(
   cb: () => void,
 ): Promise<void> {
   for (const wait of [0, 10, 100, 500, 1000]) {
+    collectGarbage();
```

Comment:
1 GC not enough?

Reply:
Surprisingly, it often isn't. When doing memory leak tests in Lit we found that memory is often not freed unless you call gc twice in a row.

🤷